zst-volatile-macro/src/lib.rs

```diff
@@ -49,3 +55,35 @@ pub fn derive_volatile(tts: TokenStream) -> TokenStream {
     }
     .into()
 }
+
+fn parse_repr(attrs: &[Attribute]) -> Result<bool, &'static str> {
```

Comment:
This function should return `syn::Result`.

Reply:
Got it, thank you!